tests/mmtf_tests.cpp

```diff
@@ -915,6 +915,12 @@ TEST_CASE("Test export_helpers") {
   REQUIRE(sd_ref.groupList == sd.groupList);
 }
 
+TEST_CASE("Test mapdecoder from raw mmtf") {
+	std::string working_mmtf = "../mmtf_spec/test-suite/mmtf/173D.mmtf";
+	mmtf::MapDecoder md(working_mmtf);
+	std::vector<int> bonds;
+	md.decode("bondAtomList", true, bonds);
```

Comment:
If you just want to check that it doesn't throw an exception, I would prefer the following:
```suggestion
	REQUIRE_NOTHROW(md.decode("bondAtomList", true, bonds));
```
Or otherwise some check of correct reading of single fields would be desired. E.g. read `numAtoms` and `xCoordList` and make sure that `xCoordList.size() == numAtoms`

Reply:
whoops! That's what I meant to do